workers/tasks/import/bulk/README.md

```diff
@@ -456,20 +456,8 @@ _The remainder of the options map directly to attributes on the Pub model._
 
 **`avatar: string`**
 
-**`headerStyle: string`**
```

Comment:
We don't have a replacement for these yet, but I don't think we've ever actually used them in the bulk importer.